GVRf/Framework/framework/src/main/jni/engine/renderer/batch.h

```diff
@@ -0,0 +1,94 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef BATCH_H
+#define BATCH_H
+#include "renderer.h"
+#include <map>
+#include <memory>
+#include <vector>
+#include <string>
+#include <set>
+namespace gvr{
+
+class RenderData;
+class Material;
+class Mesh;
+class Batch {
+public:
```

Comment:
This is all-tabs. Please get your formatter configured according to the coding style.